R/onLoad.R

```diff
@@ -0,0 +1,5 @@
+.onLoad <- function(libname, pkgname) {
+  if (is.null(getOption("rsconnect.max.bundle.size"))) {
+    options(rsconnect.max.bundle.size = 1048576000)
```

Comment:
Does/will that exist on Connect as well? Or is that purely a shinyapps.io construct?